MovingBlocks/Terasology
# feat: Upgrade Netty to version 4

### Contains

Just Upgrading netty to version 4.

### How to test

1. Start Client:
1.1. Check Singleplayer (create/load game and a bit dig)
1.2  Check Client Hosting (create/load game and a bit dig)
2. Start Server:
3. Open Multiplayer on Client.
3.1 before joining to server check request info from server (localhost)
4 Join on server and a bit dig..
